### ISSUE TITLE
Tests did not run on Travis.

### ISSUE THREAD

**Issue by moteus** [lipp/lua-websockets#57]

**Comment by lipp** [comment]
which tests?

**Comment by moteus** [comment]
I think no one 
```
/usr/local/bin/busted_bootstrap:49: attempt to call a nil value
```
And may be you can move `sudo luarocks make rockspecs/lua-websockets-scm-1.rockspec`
to install section to make more clear output.

**Comment by lipp** [comment]
Oh Dear. This is bad... seems like busted 1.11.1 is broken on travis to (maybe updates luarocks).
I'll fork busted to fix the startscript

**Comment by lipp** [comment]
fixed in #71
